### PR TITLE
More app coverage

### PR DESCRIPTION
**Pull request by cavis**
Add more apps/browsers, including:

```
Audiobooks
Castamatic
Deezer
Hamro Patro
HondaLink
Hubhopper
Instacast
KERA App
Kids Listen
Kodi
MusicBee
Orange Radio
Opera
Outcast
Playapod
Plex
PRI App
WUBR App
```

Also, updates some regexps that have changed.  Had to dig down the list of agents quite a ways to find some of these - some of them get decent traffic, but have so many app versions that grouping by user_agent string doesn't bring them to the top.

### CODE REVIEW

**Comment by cavis** [comment]
TODO: still not sure what this "Podcast" (singular) app is about, or if these are even all the same app.  They're not "The Podcast App", which I just downloaded and tried, as it has a lowercase `podcast` user agent.

```
Podcast/1.1.5 (com.nomadicdigitals.podster; build:7; iOS 12.3.1) Alamofire/4.8.1
Podcast/1150.47 CFNetwork/811.5.4 Darwin/16.6.0
Podcast/1400.149.10 CFNetwork/1082.3.1 Darwin/19.0.0
Podcast/1410.10 CFNetwork/1103 Darwin/19.0.0 (x86_64)
Podcast/36 CFNetwork/978.0.7 Darwin/18.6.0
```

Also wondering what to do with this - bot?

```
Podtrac Network Client/1.0 (beta)
```

**Comment by cavis** [comment]
Can't narrow down what app those `Podcast/` agents are coming from.  So going to ignore them for now.

That Podtrac client actually gets flagged as an AWS datacenter.  So I'm not going to worry about it for now - since IP filtering is going to flag it.

File: db/agents.yml [hunk 1]
```diff
@@ -5,12 +5,13 @@ agents:
     os: Windows
     examples:
       - +hermespod.com/v1.5.x
-  - regex: '^Acast.+Android'
+  - regex: '^Acast.+[aA]ndroid'
```
Comment (cavis) [inline db/agents.yml]: Seeing both versions in the log now.

File: db/agents.yml [hunk 2]
```diff
@@ -159,12 +175,37 @@ agents:
...
+    os: Android
+    examples:
+      - Deezer/6.0.3.44 (Android; 7.0; Mobile; us) Sony G3311
+  - regex: '^Deezer.+CFNetwork'
+    name: Deezer
+    type: Mobile App
+    os: iOS
+    examples:
+      - Deezer/6.20.0.4 CFNetwork/976 Darwin/18.2.0
+  - regex: '^(Mozilla.+)?DoggCatcher'
```
Comment (cavis) [inline db/agents.yml]: Some versions no longer include "Mozilla"

File: db/agents.yml [hunk 3]
```diff
@@ -348,18 +465,25 @@ agents:
     os: Android
     examples:
       - LibVLC/2.0.1 (Linux; U; Android 8.0.0;dreamlte/R16NW)
-  - regex: '^Luminary.+iOS'
+  - regex: '^Luminary.+(iOS|CFNetwork)'
```
Comment (cavis) [inline db/agents.yml]: Changed slightly.

File: db/agents.json [hunk 4]
```diff
@@ -638,6 +876,16 @@
         "Player FM"
       ]
     },
+    {
+      "regex": "^Plex.+iOS",
+      "name": "Plex",
+      "type": "Mobile App",
+      "os": "iOS",
```
Comment (sandikbarr) [inline db/agents.json]: if we need more OS examples, I can hit Plex pretty easily from an Apple TV, a browser, or a Roku.

File: db/agents.json [hunk 5]
```diff
@@ -1018,6 +1294,16 @@
         "com.wbez.app/7.2.2 (Linux;Android 8.0.0) ExoPlayerLib/2.5.1"
       ]
     },
+    {
+      "regex": "^W(BU|UB)R.+Android",
```
Comment (kookster) [inline db/agents.json]: oops

File: db/agents.lock.yml [hunk 6]
```diff
@@ -579,7 +705,7 @@ agents:
   - regex: bot|spider|crawl|slurp|scan|scrap|archiver|transcoder|^curl|wget|^ruby|^python|^java|perl|php|httpclient|http-client|wordpress|facebook|yahoo|^pinterest|HWCDN|appengine|hwcdn|httrack
     ignorecase: true
     bot: true
-  - regex: ^fyyd-poll|^itms|^mozilla\/5.0.+google-podcast|^stitcherbot|^rest-client|castfeedvalidator|^amazonnewscontentservice|^trackable
+  - regex: ^fyyd-poll|^itms|^mozilla\/5.0.+google-podcast|^stitcherbot|^rest-client|castfeedvalidator|^amazonnewscontentservice|^trackable|luminary\/1\.0|spotify\/1\.0
```
Comment (kookster) [inline db/agents.lock.yml]: I wonder if we ought to start tracking this distinctly?

**Pull request merged**